src/cdk/drag-drop/directives/drag.zone.spec.ts

```diff
@@ -0,0 +1,158 @@
+import {CdkScrollableModule} from '@angular/cdk/scrolling';
+import {dispatchMouseEvent} from '@angular/cdk/testing/private';
+import {
+  Component,
+  ElementRef,
+  NgZone,
+  Provider,
+  Type,
+  ViewChild,
+  ViewEncapsulation,
+  provideZoneChangeDetection,
+} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {DragDropModule} from '../drag-drop-module';
+import {Point} from '../drag-ref';
+import {CDK_DRAG_CONFIG, DragDropConfig} from './config';
+import {CdkDrag} from './drag';
+
+describe('CdkDrag Zone.js integration', () => {
```

Comment:
Could we keep this in the other file, but as a separate top-level `describe`? That way we won't have to copy the utilities.

Reply:
I prefer to extract them out to make it easier to lint against zone stuff in our tests, even if it does require a little bit of test code duplication